BaragonUI/app/components/albs/Albs.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import rootComponent from '../../rootComponent';
+import { refresh } from '../../actions/ui/albs';
+
+import Utils from '../../utils';
+
+import TargetGroupsTable from './TargetGroupsTable';
+import LoadBalancersTable from './LoadBalancersTable';
+
+const Albs = ({targetGroups, loadBalancers}) => {
+  return (
+    <div>
+      <h3>Application Load Balancers</h3>
+      <div className="row">
+        <div className="col-md-12">
+          <h4>Load Balancers</h4>
+          <LoadBalancersTable loadBalancers={loadBalancers} />
+        </div>
+      </div>
+      <div className="row">
+        <div className="col-md-12">
+          <h4>Target Groups</h4>
+          <TargetGroupsTable targetGroups={targetGroups} />
+        </div>
+      </div>
+    </div>
+  );
+};
+
+Albs.propTypes = {
+  targetGroups: PropTypes.array,
+  loadBalancers: PropTypes.array,
+};
+
+const mapStateToProps = (state) => ({
+  loadBalancers: Utils.maybe(state, ['api', 'loadBalancers', 'data']),
```

Comment:
shouldn't need the `maybe` here. Since you are setting the default in the reducers index, we know `state.api.loadBalancers.data` and `state.api.targetGroups.data` should be present. Might be empty, but will be there.